vendor/github.com/spf13/cobra/command.go

```diff
@@ -994,7 +1030,11 @@ func (c *Command) InitDefaultVersionFlag() {
 		} else {
 			usage += c.Name()
 		}
-		c.Flags().Bool("version", false, usage)
+		if c.Flags().ShorthandLookup("v") == nil {
+			c.Flags().BoolP("version", "v", false, usage)
```

Comment:
does this cause any issues with our usage of `-v` to control log verbosity?

Reply:
test with `-v` locally, no issue happened.